keps/sig-api-machinery/1965-kube-apiserver-identity/README.md

```diff
@@ -265,7 +261,7 @@ the Lease object to see if workloads or other controllers are relying on this fe
 ###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
 Some reasonable SLOs could be:
-* Number of (non-expired) Leases in `kube-apiserver-leases` is equal to the number of expected kube-apiservers 95% of the time.
+* Number of (non-expired) Leases in `kube-system` is equal to the number of expected kube-apiservers 95% of the time.
```

Comment:
are the kube-apiserver leases named (or ideally labeled) in a way that makes them easily findable?